app/helpers/colors_helper.rb

```diff
@@ -61,6 +68,21 @@ def colored_text(color)
     content_tag(:span, color.hexcode, class: 'color--text-preview', style: style)
   end
 
+  #
+  # Styles to display colors itself (e.g. for the colors autocompleter)
+  ##
+  def color_css
+    Color.find_each do |color|
+      concat ".__hl_inline_color_#{color.id}_dot::before { background-color: #{color.hexcode} !important;}"
+      concat ".__hl_inline_color_#{color.id}_dot::before { border: 1px solid #555555 !important;}" if color.bright?
+      concat ".__hl_inline_color_#{color.id}_text { color: #{color.hexcode} !important;}"
+      concat ".__hl_inline_color_#{color.id}_text { -webkit-text-stroke: 0.5px grey;}" if color.super_bright?
```

Comment:
You should probably set `text-stroke: ` as well as per https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke